ehitchhiking-ui/src/app/shared/components/rate-passangers-modal/rate-passangers-modal.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="table-content" >
+    <div class='container'>
+      <h2 class="title">Rate Passangers</h2>
+      <div class="panel panel-primary">
+        <div class="table-responsive">
+          <table class="table">
+            <thead>
+            <tr class="table-tr">
+              <th>Passanger</th>
+              <th>Stars</th>
+              <th>Rating</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr *ngFor="let item of items">
```

Comment:
track by id

Reply:
item  - bad naming. It is a user not item

Reply:
> track by id

not understood

Reply:
https://netbasal.com/angular-2-improve-performance-with-trackby-cc147b5104e5
in your case will be *ngFor="let item of items; trackBy: item.id"